Description: Change the default font to one very likely to be present on a
 Debian box.

Index: debian.git/modules/misc/freetype.c
===================================================================
--- debian.git.orig/modules/misc/freetype.c	2010-06-16 01:58:09.000000000 +0200
+++ debian.git/modules/misc/freetype.c	2010-06-23 00:46:55.485476112 +0200
@@ -67,7 +67,7 @@
 #define DEFAULT_FONT "/usr/share/fonts/nokia/nosnb.ttf"
 #define FC_DEFAULT_FONT "Nokia Sans Bold"
 #else
-#define DEFAULT_FONT "/usr/share/fonts/truetype/freefont/FreeSerifBold.ttf"
+#define DEFAULT_FONT "/usr/share/fonts/truetype/ttf-dejavu/DejaVuSans-Bold.ttf"
 #define FC_DEFAULT_FONT "Serif Bold"
 #endif
 
